malformed/test_chunking_example.py

```diff
@@ -0,0 +1,235 @@
+            "GET / HTTP/1.1\r\n" \
+            "User-Agent: Wget/1.13.4 (linux-gnu)\r\n" \
+            "Accept: */*\r\n" \
+            "Host: localhost\r\n" \
+            "Connection: Keep-Alive\r\n" \
+            "X-Custom-Hdr: custom header values\r\n" \
+            "Dummy0: 0\r\n" \
+            "Dummy1: 1\r\n" \
+            "Dummy2: 2\r\n" \
+            "Dummy3: 3\r\n" \
+            "Dummy4: 4\r\n" \
+            "Dummy5: 5\r\n" \
+            "Dummy6: 6\r\n" \
+            "Content-Type: text/html; charset=iso-8859-1\r\n" \
+            "Dummy7: 7\r\n" \
+            "Dummy8: 8\r\n" \
+            "Dummy9: 9\r\n" \
+            "Cache-Control: max-age=1, no-store, min-fresh=30\r\n" \
+            "Pragma: no-cache, fooo \r\n" \
+            "Transfer-Encoding: compress, gzip, chunked\r\n" \
+            "Cookie: session=42; theme=dark\r\n" \
+            "Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==\t \n" \
+            "\r\n" \
+            "6\r\n" \
+            "123456\r\n" \
+            "0\r\n" \
+            "\r\n"
+            # Excluded: "X-Forwarded-For: 127.0.0.1, example.com\r\n"
+            # because TFW rewrites it
+        self.iterate_test(self.inner_test_correct_headers, request)
+
+    def inner_test_ss_chunks(self, chunksize, request):
+        # This function makes a simple request to check
+        # (with tcpdump) that server response is chunked
+        # as required
+        deproxy_srv = self.get_server('deproxy')
+        # This line controls chunking:
+        deproxy_srv.segment_size = chunksize
+        deproxy_srv.start()
+        self.start_tempesta()
+        deproxy_cl = self.get_client('deproxy')
+        deproxy_cl.start()
+        self.deproxy_manager.start()
+        self.assertTrue(deproxy_srv.wait_for_connections(timeout=1))
+        deproxy_cl.make_request(request)
+        self.assertTrue(deproxy_cl.valid_req_num != 0,
+                "Request was not parsed by deproxy client")
+        has_resp = deproxy_cl.wait_for_response(timeout=5)
+        self.assertTrue(has_resp, "Response not received"
+                        + "; with chunk size = " + str(chunksize))
+        status = int(deproxy_cl.last_response.status)
+        self.assertTrue(status == 200, "Wrong status: " + str(status)
+                                        +  ", expected: 200"
+                        + "; with chunk size = " + str(chunksize))
+        self.assertFalse(deproxy_srv.last_request is None,
+                           "Request was not send to backend"
+                        + "; with chunk size = " + str(chunksize))
+
+    def test_ss_chunks(self):
+        # This function makes a simple request to check
+        # (with tcpdump) that server response is chunked
+        # as required, iterating over various chunk sizes
+        request = \
+            "GET / HTTP/1.1\r\n" \
+            "Host: localhost\r\n" \
+            "\r\n"
+        self.iterate_test(self.inner_test_ss_chunks, request)
```

Comment:
This looks like the test from https://github.com/tempesta-tech/tempesta/issues/810#issuecomment-966690711 "Chunked responses as requested by #1535", so
1. it makes sense to move the test to some more meaningful place (at the moment it's in "tests for a test" or "examples for a test")
2. I couldn't quickly find how large the deproxy response is, but it should be at least several kilobytes.

Reply:
Made separate test